.travis.yml

```diff
@@ -3,12 +3,6 @@ language: php
 dist: precise
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
-    - 7.1
```

Comment:
Security support for PHP 7.1 ends on December 1, 2019:

<img width="817" alt="Screen Shot 2019-11-14 at 15 33 09" src="https://user-images.githubusercontent.com/605483/68866045-1ad25400-06f4-11ea-97e6-e003dcc69830.png">